### ISSUE TITLE
Documentation Error for cval Parameter in denoise_bilateral Function

### ISSUE THREAD

**Issue by tokiAi** [scikit-image/scikit-image#7247]
### Description:

https://github.com/scikit-image/scikit-image/blob/main/skimage/restoration/_denoise.py#L94

`def denoise_bilateral(image, win_size=None, sigma_color=None, sigma_spatial=1, bins=10000, mode='constant', cval=0, *, channel_axis=None,):`

```
Parameters
    ----------
    ...
    cval : string
        Used in conjunction with mode 'constant', the value outside
        the image boundaries.
```

There appears to be an inconsistency in the documentation of the `denoise_bilateral` function regarding the `cval` parameter. The `cval` parameter is described as a `string` in the documentation, which seems incorrect. Typically, `cval` is a numeric value that specifies the constant value used for padding (when mode is 'constant').

#### Suggested Correction

It would be more appropriate if `cval` is described as a numeric value (e.g., `int` or `float`) instead of a string. This change would align with the typical usage of `cval` in similar contexts where it represents a constant padding value. 

**Is `cval` a `int` or `float`?**

**Comment by soupault** [comment]
Thanks! The argument is converted to float in the underlying call - https://github.com/scikit-image/scikit-image/blob/main/skimage/restoration/_denoise_cy.pyx#L25 . I would be fine with either "float" or "int or float".

**Comment by soupault** [comment]
Fixed in #7319.
